### ISSUE TITLE
SystemStackError stack level too deep

### ISSUE THREAD

**Issue by DanielViglione** [ankane/chartkick#133]
Using the line_chart throws an exception in Rails "stack level too deep". My data looks like this:

[{:name=>:unique_call, :data=>#<Groupdate::Series:0x007faf48edddd0 @magic=#<Groupdate::Magic:0x007faf48f1d278 @field=:week, @options={}, @time_zone=#<ActiveSupport::TimeZone:0x007faf453b4f88 @name="Eastern Time (US & Canada)", @utc_offset=nil, @tzinfo=#<TZInfo::TimezoneProxy: America/New_York>, @current_period=#<TZInfo::TimezonePeriod: #<TZInfo::TimezoneTransitionDefinition: #<TZInfo::TimeOrDateTime: 1394348400>,#<TZInfo::TimezoneOffset: -18000,3600,EDT>>,#<TZInfo::TimezoneTransitionDefinition: #<TZInfo::TimeOrDateTime: 1414908000>,#<TZInfo::TimezoneOffset: -18000,0,EST>>>>, @week_start=6, @day_start=0, @time_range=nil, @group_index=0>, @relation={nil=>2}>}]

Basically I followed this technique:

<%= line_chart @goals.map{|goal|
    {name: goal.name, data: goal.feats.group_by_week(:created_at).count}
} %>

except mine looks like this:

@chart_data = []
@chart_data << { name: :unique_call, data: leads.group_by_week(:created_at).size }
line_chart @chart_data

Why I am getting this error with the line chart? I don't get errors with the pie chart.
